### PR TITLE
DOC: Tweak comments and docstrings

### PR DESCRIPTION
**Pull request by j-t-1**
Small changes to comments and docstrings in class PdfWriter.

### CODE REVIEW

File: pypdf/_writer.py [hunk 1]
```diff
@@ -398,7 +398,7 @@ def _repr_mimebundle_(
         This method returns a dictionary that maps a mime-type to its
         representation.
 
-        See https://ipython.readthedocs.io/en/stable/config/integrating.html
+        https://ipython.readthedocs.io/en/stable/config/integrating.html
```
Comment (stefan6419846) [inline pypdf/_writer.py:401]: Could you please elaborate why you consider this change necessary?

Comment (j-t-1) [reply pypdf/_writer.py:401]: It is superfluous to the intent of the link. Feel free to add back.

Comment (stefan6419846) [reply pypdf/_writer.py:401]: I would probably go with the `seealso` directive of Sphinx here.

Comment (stefan6419846) [reply pypdf/_writer.py]: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-seealso

**Review by stefan6419846 — APPROVED** [review]

**Pull request merged**